pkg/compact/compact_test.go

```diff
@@ -91,45 +91,6 @@ func TestRetryError(t *testing.T) {
 	testutil.Assert(t, IsHaltError(err), "not a halt error. Retry should not hide halt error")
 }
 
-func TestGroupKey(t *testing.T) {
```

Comment:
Groupkey depends on the labels implementation, slicelabels have a different one then stringlabels.